src/Marquee/Marquee.js

```diff
@@ -950,7 +950,7 @@ var MarqueeItem = {
 		var node = this.$.marqueeText ? this.$.marqueeText.hasNode() : this.hasNode(),
 			rect;
 
-		if (node && this._marquee_distance == null && this.getAbsoluteShowing()) {
+		if (node && this.getAbsoluteShowing()) {
```

Comment:
Where you say "always re-measures" I say "consistently re-measures". :-) Is it really that much of a performance hit? I was thinking of calling reflow() on the marquee but don't know the best way to trigger that given that ItemOverlay uses CSS rules to toggle.